docs/docs/installation.md

```diff
@@ -75,3 +69,12 @@ task build
 ```
 
 After this the binary is available on path `./bin/jalapeno`.
+
+## Updating your jalapeno version
```

Comment:
In documentation should we use capitalised Jalapeno?

Reply:
Fixed in [d59670c](https://github.com/futurice/jalapeno/pull/48/commits/d59670cd6b1f71c57fad05e5de0656aaa4266023)